Assets/LeapMotion/Scripts/Utils/Utils.cs

```diff
@@ -167,6 +167,34 @@ public static class Utils {
 
     #endregion
 
+    #region Collider Utils
+
+    public static Vector3 GetDirection(this CapsuleCollider capsule) {
+      return capsule.direction == 0 ? Vector3.right : capsule.direction == 1 ? Vector3.up : Vector3.forward;
```

Comment:
Could we use a switch instead of nested ternary statements?  I think it would really help readability.